packages/paste-website/src/pages/form/input/index.mdx

```diff
@@ -0,0 +1,578 @@
+- **Optional field indicator** — In a form where there are more required fields than optional fields, use an optional indicator to show users which fields they don’t have to fill out. In general, ask for information only when needed. Consider removing them otherwise.
+- **Help text** — Text that’s placed below the field to help users prevent an error and describe what makes the form field successful.
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="email_address" required>Email Address</FormLabel>
+  <FormInput id="email_address" type="email" placeholder="example@twilio.com" required />
+  <FormHelperText>Please enter a valid email</FormHelperText>
+</>`}
+</LivePreview>
+
+### Input with add-ons (prefix/suffix text or icons)
+
+- **Prefix/suffix text** — Text that can be used as a prefix and/or suffix to the value that is entered. Use prefix/suffix to help users format text.
+- **Icon** — Icons can be placed in the same area as the prefix and suffix text. Icons should trigger an action (_e.g._, clearing a field) or in rare cases, provide further context to what value should be entered to make a field’s purpose more immediately visible (_e.g._, a search icon).
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="payment_amount" required>Payment Amount</FormLabel>
+  <FormInput id="payment_amount" type="text" placeholder="124.45" insertBefore={<div>$</div>} required />
+  <FormHelperText>Enter a dollar amount is USD format</FormHelperText>
+</>`}
+</LivePreview>
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText, Anchor, InformationIcon}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="message_title" required>Message Title</FormLabel>
+  <FormInput
+    id="display_name"
+    type="text"
+    placeholder="Ahoy, World"
+    insertAfter={
+      <Anchor href="#" display="flex">
+        <InformationIcon decorative={false} title="Get more info" />
+      </Anchor>
+    }
+  />
+  <FormHelperText>Please enter a short title for your message</FormHelperText>
+</>`}
+</LivePreview>
+
+### Input types
+
+How an input functions varies depending on the value of its type attribute. If a type attribute is not specified, the default type is text. The following type options are available:
+
+#### text
+
+- A single-line text input
+- Line-breaks are automatically removed from the input value
+- Applies type=”text” by default
+
+#### email
+
+- An input for editing an email address
+- Looks like a text input, but has validation parameters and relevant keyboard in supporting browsers and devices with dynamic keyboards
+
+#### hidden
+
+- An input that is not displayed but whose value is submitted to the server
+
+#### number
+
+- An input for entering a number
+- Displays a stepper (small up/down arrows) and adds default validation when supported
```

Comment:
Are these 2 true now that we don’t use the native number type?

Reply:
What're we using? The thing from that post Shadi shared?

Reply:
Right now, we are allowing a user to have a number input. Should we remove that option from the types?

Reply:
I think we still need some input that restricts the value to numbers (however it's implemented), especially since it's used in all the products.

If we don't use the native number type, do we need to mock up the stepper? Is it a pretty significant chunk of eng work to build the number input?